lldb/test/API/macosx/unwind-frameless-faulted/Makefile

```diff
@@ -0,0 +1,12 @@
+C_SOURCES := main.c
+
+interrupt-and-trap-funcs.o: interrupt-and-trap-funcs.s
+	$(CC) $(CFLAGS) -c -o interrupt-and-trap-funcs.o $(SRCDIR)/interrupt-and-trap-funcs.s
```

Comment:
Should this pass a triple? I assume this works locally because you're on an arm64 host?